examples/react-webpack/src/App.js

```diff
@@ -1,11 +1,21 @@
 import './superhero-utils/index.css';
 import { Button as SuperheroButton } from './superhero-utils/react-without-styles.js';
 
+import './style.css';
+
 function App() {
   return (
-    <div>
+    <div className="Example-List">
       <h3>superhero-utils example using react, webpack</h3>
+      medium:
       <SuperheroButton size="medium" account="ak_... or .chain name" />
+
+      {['large', 'small', 'icon'].map(size =>
```

Comment:
I think the initial example is fine, and we shouldn't spend any more time working on this. The purpose of building these examples was to ensure that our button is compatible with different use cases.

Reply:
I think that until we don't have any tests, it will be useful to run render as much possible variances as we can, to make it easier to check if something is wrong.

Reply:
Let's combine it with something useful I'm proposing to build a code generator previewing the current state (#18). But before doing that you need to ensure that you have **approval** for that. cc: @mradkov